src/locales/en/pages.yml

```diff
@@ -59,8 +59,7 @@ locations:
   body: "Postcode"
 
 photoIdExpiry:
-  pageTitle: "Your photo ID has expired"
   title: "Your photo ID has expired"
-  one: "We can only accept photo IDs that are currently in date. Expired UK passports are acceptable up to 18 months after the expiry date."
-  two: "If you entered the wrong expiry date, go back to enter the correct date."
-  three: "To select another valid photo ID, click Continue."
+  h1: "Your photo ID has expired"
+  content: "You need a valid ID to prove your identity.
+            <br><br>"
```

Comment:
is this needed?

Reply:
Yep this sets the content for the page above the radio form